products/BellHybrid/apps/application-bell-settings/models/DateTimeUnitsModel.cpp

```diff
@@ -62,6 +62,7 @@ namespace app::bell_settings
         timeSetListItem =
             new gui::TimeSetListItem(0U, 0U, 0, 0, utils::translate("app_bell_settings_time_units_time_message"));
         internalData.push_back(timeSetListItem);
+        timeSetListItem->getTimeFormat = [this](void) { return timeFmtSetListItem->getTimeFmt(); };
```

Comment:
Why have you resolved my comment in your previous PR without even a word of explanation? If I'm mistaken in my suggestion or in your opinion the change is irrelevant just post a reply with an explanation instead of just resolving the conversation and merging the PR anyway.

I'm aware that this change is a marginally important detail, but what's the point of requesting the reviews if the code is merged despite the suggestions.

Reply:
Alright, I get it :wink: Let's leave it as it is to not introduce any discrepancies between master and release branch